tests/Assets/scripts/postgres/01_custom_event_streams_table.sql

```diff
@@ -0,0 +1,12 @@
+CREATE SCHEMA IF NOT EXISTS custom;
```

Comment:
I don't know if "custom" is a good default, why not "prooph" :smile:

Reply:
ready